services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/entity/decider/UsersEntityAccessInfo.java

```diff
@@ -28,7 +29,7 @@ public UsersEntityAccessInfo(AccessContext context, AuthorizationStatus status)
 		this.entityId = context.getPermissionsState().getEntityId();
 		this.benefactorId = context.getPermissionsState().getBenefactorId();
 		this.entityExists= context.getPermissionsState().doesEntityExist();
-		this.accessRestrictions = context.getRestrictionStatus();
+		this.userRestrictionStatusWithHasUnmet = context.getRestrictionStatusWithHasUnmet();
 		this.authorizationStatus = status;
```

Comment:
This class will use the static util to capture the boolean, with no other changes.